docs/earthfile/earthfile.md

```diff
@@ -1371,6 +1371,16 @@ The `USER` command sets the user name (or UID) and optionally the user group (or
 
 The `WORKDIR` command `strs` the working directory for other commands that follow in the recipe. The working directory is also persisted as the default directory for the image. If the directory does not exist, it is automatically created. This command works the same way as the [Dockerfile `WORKDIR` command](https://docs.docker.com/engine/reference/builder/#workdir).
```

Comment:
offtopic: what is `strs`?

Reply:
Not sure...?

Reply:
here's an update for that: https://github.com/earthly/earthly/pull/1798